modules/distr-tracing-tempo-install-web-console.adoc

```diff
@@ -36,7 +36,9 @@ This installs the Operator with the default presets:
 
 .. In the *Details* tab of the page of the installed Operator, under *ClusterServiceVersion details*, verify that the installation *Status* is *Succeeded*.
 
-. Create a secret for your object storage bucket: go to *Workloads* -> *Secrets* -> *Create* -> *From YAML*.
+. Create a project of your choice for the *TempoStack* instance that you will create in a subsequent step: go to *Home* -> *Projects* -> *Create Project*.
```

Comment:
I realize that it was worded mostly like this before you moved it, so feel free to ignore. But I have two suggestions:
* I don't think it's necessary to say "that you will create in a subsequent step". A lot of steps in procedures are needed for things later in the procedure.
* I'm not a fan of the sentence explanation and then a colon and then the actual steps in the UI. I think it might look better if the UI steps would be in a subbullet under it

Same applies for any of the other steps in the PR that apply - I won't point out any other instances

Reply:
Thank you for your suggestion. In this case, `the *TempoStack* instance` does not exist yet, so `that you will create in a subsequent step` explains when it will appear.